src/tests/createHeadlessForm.test.js

```diff
@@ -1442,6 +1443,45 @@ describe('createHeadlessForm', () => {
       });
     });
 
+    it('supports oneOf pattern validation', () => {
+      const result = createHeadlessForm(
+        JSONSchemaBuilder()
```

Comment:
todo: `JSONSchemaBuilder` is deprecated, could you just use normal JSON?

related: #53